package.json

```diff
@@ -14,6 +14,7 @@
     "build/",
     "doc/",
     "lib/",
+    "patches0/",
```

Comment:
My bad to have forgotten to add this line here. I had tested the release assets of PR #1739 (standalone zip packages, macOS installer, Windows installer) but I forgot to test `npm install -g --production`. Luckily the consequence is limited to the warning message during the installation.